server/views/formPages/incidentMacros.njk

```diff
@@ -36,7 +36,15 @@
 </div>
 {% endmacro %}
 
+
 {% macro radiosWithNestedRadios(question) %}
+
+{% if question.orientation === 'inline'  %}
+{% set radio_button_orientation = 'govuk-radios--inline'%}
+{% else %}
+{% set radio_button_orientation = 'govuk-radios' %}
```

Comment:
would have hoped that if this adhered to the standard GDS pattern, then we wouldn't have to use (and so adapt) our custom macro for this case..

Reply:
done this way